frontend/app/router.js

```diff
@@ -16,12 +16,13 @@ const Router = EmberRouter.extend({
   metrics: service(),
 
   _trackPage() {
-    scheduleOnce('afterRender', this, () => {
+    function trackPage() {
       const page = this.get('url');
       const title = this.getWithDefault('currentRouteName', 'unknown');
 
       this.metrics.trackPage({ page, title });
-    });
+    }
+    scheduleOnce('afterRender', this, trackPage);
```

Comment:
Is this the result of a codemod or something? Or downstream of upgrading ember?

Reply:
This was an eslint thing that was suggested. Wouldn't let me commit before I changed it. 

https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-incorrect-calls-with-inline-anonymous-functions.md

Reply:
Ah, okidoke! I see, the ember upgrade modified the file (removed line 104-105).

Reply:
Interesting, reading the lint rule, it sounds like with the anonymous function it'd run twice?  I think that this change also improves the readability:
` scheduleOnce('afterRender', this, trackPage);`
I like this because it reads more declaratively (you could then go to trackPackage definition to see how it's working, but this line declares _what_ happens.